modules/data-entry/src/main/frontend/src/questionnaire/DateQuestionFull.jsx

```diff
@@ -152,9 +152,18 @@ function DateQuestionFull(props) {
     )
   }
 
+  let displayFormatter = function(label, idx, value) {
+    return (
+      <Tooltip title={value}>
```

Comment:
Access the raw value directly via `idx`, not tested:

```suggestion
  let displayFormatter = function(label, idx) {
    return (
      <Tooltip title={Array.from(startValues || [])[idx]}>
```

Reply:
fixed in [c0bf7ec](https://github.com/data-team-uhn/cards/pull/1879/commits/c0bf7ecabcf26671bb04cd5f532c9b76ba889180)